auth.css

```diff
@@ -0,0 +1,127 @@
+@import "base.css";
+
```

Comment:
I don't know what CSS file this is, please make your CSS name clear.
You should have a login.html and login.css
And also a register.html and register.css